static/src/server.ts

```diff
@@ -457,6 +461,7 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
   const startTime = process.hrtime.bigint();
   const query = req.query.q;
   const useChartUrl = req.query.chartUrl !== CHART_URL_PARAM_SVG;
+  const allResults = req.query.all === SHOW_ALL_URL_PARAM;
```

Comment:
nit:  if we have a param, should we do maxResults?  Default = 3, to disable set 0.

if you prefer a bool, have a more descriptive param name?

Reply:
I prefered a bool so updated the param name to be `allCharts`